modules/benchmark/R/calc_benchmark.R

```diff
@@ -18,24 +18,50 @@ calc.benchmark <- function(settings, bety) {
   collect <- dplyr::collect
   select  <- dplyr::select
   
+  # --------------------------------------------------------------------------------------------- #
+  # Update benchmarks_ensembles and benchmarks_ensembles_scores tables
+  
+  ensemble <- tbl(bety,'ensembles') %>% filter(workflow_id == settings$workflow$id) %>% collect()
+  
+  # Retrieve/create benchmark ensemble database record
+  bm.ensemble <- tbl(bety,'benchmarks_ensembles') %>% 
+    filter(reference_run_id == settings$benchmarking$reference_run_id) %>% 
+    filter(ensemble_id == ensemble$id) %>% 
+    filter(model_id == settings$model$id) %>%
+    collect()
```

Comment:
Question I've been wondering about this syntax... Does this literally grab the entirety of the database table into R and then do the filtering in R, or is it using these function calls to build the SQL query that it then executes once?   If the former, this seems really inefficient. If the latter, do the rest of you really find this syntax easier to follow than SQL? I find it much harder to understand.  Not requesting a change (yet), just trying to understand memory impacts and get a feel for how this works.